lib/config.py

```diff
@@ -62,15 +62,19 @@
         dict(help='Branch of the release notes repository to checkout'),
 }
 PUSH_REPO_ARGS = {
+    ('--commit-updates',):
+        dict(help='Commit file updates to local repository'),
+    ('--push-updates',):
+        dict(help='Push file updates to remote repository'),
```

Comment:
Those parameters will require a string argument, and the resulting variable's value will always evaluate to `True`.
Since the default we want is the boolean value `True`, and we only need the parameters to disable those actions, we should use `action='store_false'` and rename the parameters to something like `--no-commit-updates` and `--no-push-updates`.
If you really wanted to keep the parameter names, you could pass `type=ast.literal_eval`, so `--commit-updates False` would set the variable to the bolean `False`, but I don't think this is a good approach.